src/reducers/testReducer.js

```diff
@@ -0,0 +1,9 @@
+const testReducer = ( state=[], action ) => {
+  switch(action.type) {
+    case "TEST_ACTION":
+      return console.log("Testing stuff")
```

Comment:
(Part Four)

So then everything looks good, right? No. It blows up. Because `console.log()` returns undefined and we're returning `console.log()` instead of the updated state.

```js
const testReducer = ( state= [], action ) => {
  switch(action.type) {
    case "TEST_ACTION":
      console.log("Testing stuff")
      return state;
  }
  return state
};
```

Let's just return the same state for now.